client/src/app/tabs/form/linting/FormLinter.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { isString } from 'min-dash';
+
+
+export default class FormLinter {
+  static lint(contents) {
+    let schema;
+
+    if (isString(contents)) {
+      schema = JSON.parse(contents);
+    } else {
+      schema = contents;
+    }
+
+    const {
+      executionPlatform,
+      executionPlatformVersion
+    } = schema;
+
+    if (!executionPlatform) {
+      return [];
+    }
+
+    const types = [
+      'button',
+      'default',
+      'textfield'
+    ];
+
+    if ((executionPlatform === 'Camunda Platform' && executionPlatformVersion === '7.16')
```

Comment:
Proposal: Let us map the execution platform to form-js versions internally.

This way we can test on the form-js versions released (i.e. v0.1, v0.3) rather than the execution platforms.

Reply:
Done (cf. https://github.com/camunda/camunda-modeler/pull/2399/commits/77bbfcf37e627e88d52a8862f3bf786d584bd248)